kibana-notebooks/public/components/notebook.tsx

```diff
@@ -455,6 +463,29 @@ export class Notebook extends Component<NotebookProps, NotebookState> {
       });
   };
 
+  checkQueryOutputError = (checkErrorJSON: JSON) => {
+    // if query output has error output
+    if (checkErrorJSON.hasOwnProperty('error')) {
+      this.setState({
+        showQueryParagraphError: true
+      });
+      this.setState({
+        queryParagraphErrorMessage: checkErrorJSON.error.reason
+      });
+      return true;
+    }
+    // query ran successfully, reset error variables if currently set to true
+    else if (this.state.showQueryParagraphError) {
+      this.setState({
+        showQueryParagraphError: false
+      });
+      this.setState({
+        queryParagraphErrorMessage: ''
```

Comment:
you can combine setstate calls instead of calling it twice `setState({ showQueryParagraphError: false, queryParagraphErrorMessage: '' })`. same for line 469